fairscale/experimental/nn/offload.py

```diff
@@ -164,10 +165,11 @@ def forward(ctx: Any, inputs: Any, model_instance: Any) -> Any:
         model_instance._activations = [inputs]
         # Enumerate through layer shards and apply activations from the previous shard.
         for index, layer_shard in enumerate(model_instance.model_slices):
-            # Bring in the current activations onto the device.
-            model_instance._activations[index] = tuple([a.cuda() for a in list(model_instance._activations[index])])
             # Bring in the current layer shard onto the device.
             layer_shard.forward_load()
+            # Bring in the current activations onto the device.
```

Comment:
any logic as of why this is flipped (layer/activation load) when compared to before ?

Reply:
I was trying out some stuff and flipped the order for readability. Flipped it back to avoid confusion.